File: exercises/beecrowd/1060/1060.go
// https://judge.beecrowd.com/pt/problems/view/1060
package main

import (
	"fmt"
)

func main() {
	var v1, v2, v3, v4, v5, v6 float64

	fmt.Scanln(&v1)
	fmt.Scanln(&v2)
	fmt.Scanln(&v3)
	fmt.Scanln(&v4)
	fmt.Scanln(&v5)
	fmt.Scanln(&v6)

	values := []float64{v1, v2, v3, v4, v5, v6}

	var positiveNumbers int

	for _, v := range values {
		if v > 0 {
			positiveNumbers++
		}
	}

	fmt.Printf("%v valores positivos\n", positiveNumbers)
}

File: exercises/beecrowd/1060/1060.ts
// https://judge.beecrowd.com/pt/problems/view/1060
import * as fs from "fs";

function main(): void {
	const input = fs.readFileSync('/dev/stdin', 'utf8');
    const lines = input.split(/\r?\n/);
    
    const v1 = Number(lines.shift());
    const v2 = Number(lines.shift());
    const v3 = Number(lines.shift());
    const v4 = Number(lines.shift());
    const v5 = Number(lines.shift());
    const v6 = Number(lines.shift());

	const values = [v1, v2, v3, v4, v5, v6];

	let positiveNumbers = 0;

	for (const v of values) {
		if (v > 0) {
			positiveNumbers++;
		}
	}

	console.log(`${positiveNumbers} valores positivos`);
}

main();
